README-devs.md

```
@@ -0,0 +1,138 @@
+✔ Container relay_nakama             Created                                                                                                                                                                    0.0s
+✔ Container test_nakama              Created                                                                                                                                                                    0.0s
+Attaching to cockroachdb, redis, relay_nakama, test_game, test_nakama
+</pre>
+
+After a short time, you will see `test_game` ticking, along with messages from `relay_nakama` and `test_nakama`:
+
+<pre>
+test_game  | 7:19PM INF Tick completed duration=1.02933ms tick=1385 tx_count=0
+</pre>
+
+If you check the logs from the `test_nakama` container ( cmd-8 for Services, then click on 
```

Markdownlint

Expected: 0 or 2; Actual: 1
Trailing spaces

(MD009, no-trailing-spaces)